### PR TITLE
Added support for binary gems.

### PR DESCRIPTION
**Pull request by haberman**

### CODE REVIEW

File: ruby/Rakefile [hunk 1]
```diff
@@ -21,6 +21,11 @@ else
     ext.ext_dir = "ext/google/protobuf_c"
     ext.lib_dir = "lib/google"
   end
+
+  task 'gem:windows' do
+    require 'rake_compiler_dock'
```
Comment (nicolasnoble) [inline ruby/Rakefile]: I think you need to add
```
s.add_development_dependency "rake-compiler-dock"
```

in the Gemfile for it to work.

Comment (haberman) [inline ruby/Rakefile]: Done.

**Comment by haberman** [comment]
@nicolasnoble would you mind confirming that you want to contribute those two changes to this repo, to make the CLA bot happy?

**Comment by nicolasnoble** [comment]
I am a googler and hence am okay with these commits being contributed to Google ;-)

**Comment by haberman** [comment]
Thanks. :)

**Pull request merged**